docs/user/agent/minimal-iso.md

```diff
@@ -42,8 +40,12 @@ spec:
 ................
 ```
 
+## Configuring a Minimal ISO on other platform types
+
+For platform types other than External, the agent-based installer can be configured to generate a minimal ISO via the `minimalISO` field in agent-config.yaml. By default this field in False; when set to True the agent-based installer will generate a minimal ISO.
+
 ## Generation of Minimal ISOs
-For the external platform, the agent-based installer always generates a minimal ISO, but may or may not generate the rootfs file explicitly. A minimal ISO is similar to the full ISO generated for other platforms, with the distinction that it does not contain the rootfs file within it.
+When the agent-based installer generates a minimal ISO it may or may not generate the rootfs file explicitly. A minimal ISO is similar to the full ISO generated for other platforms, with the distinction that it does not contain the rootfs file within it.
```

Comment:
```suggestion
When the agent-based installer generates a minimal ISO it may or may not generate the rootfs file explicitly depending whether the rootFS URL is specified via the `bootArtifactsBaseURL` field in `agent-config.yaml.` A minimal ISO is similar to the full ISO generated for other platforms, with the distinction that it does not contain the rootfs file within it.
```

Reply:
Good suggestion. Updated.